common/utils.py

```diff
@@ -0,0 +1,91 @@
+from six.moves import configparser
+import logging
+import os
+import requests
+
+
+def get_setting(config_file, config_name, setting, default=None):
+    config = configparser.SafeConfigParser()
+    try:
+        config.read(config_file)
+        return config.get(config_name, setting)
+    except Exception:
+        return default
+
+
+def configure_logging(filename): #get_setting('logfile', default_logfile)
+    CONFIG = {
+        'version': 1,
+        'disable_existing_loggers': False,
+        'formatters': {
+            'default': {
+                'format': '%(levelname)-8s  %(asctime)s  %(filename)s:%(lineno)-4s %(message)s',
+                'datefmt': '%Y-%m-%d %H:%M:%S',
+            },
+        },
+        'handlers': {
+            'console': {
+                'class': 'logging.StreamHandler',
+                'formatter': 'default',
+            },
+            'file': {
+                'class': 'logging.handlers.RotatingFileHandler',
+                'formatter': 'default',
+                'filename': filename,
+                'backupCount': 2,
+                'maxBytes': 10 * 1024,
+            },
+        },
+        'loggers': {
+            'storage': {
```

Comment:
Since this is a common util now, it should let the caller define what logger to listen for.  And since we're adding a common logger, it should be configured here by default too.